Demo.md

```diff
@@ -17,88 +17,81 @@ example, local networks.
 - `Central` is a deployment that receives messages from `nodes`, processing and
 presenting data related to them.
 
-We opted to deploy this poc as a StarlingX Application instead of deploying It directly through Kubernetes or with Helm Charts
-because our team had two major motivations:
+## How to deploy your workload
 
-1. We wanted to learn more about how to build a StarlingX Application.
-2. We want to make further improvements in the application to integrate
-   It with StarlingX sysinv to automatize specific nodes configurations
-   depending on the system states.
+We opted to deploy this proof of concept as a StarlingX Application. It's
+important to understand that any user workload can be deployed in many ways to
+the Kubernetes cluster(s) that StarlingX manages:
 
-But we would like to make It clear that if your application does not require a
-deeper integration with the StarlingX system, never needs to
-change its behaviour or have changes that depend on the user only, then,
-deploying the application directly into StarlingX "vanilla" Kubernetes,
-with Helm Charts and even FluxCD if you'd like to enjoy the benefits of CI/CD
-pipelines.
+- with the most common Kubernetes package manager, [Helm](https://helm.sh/);
+- with [Flux](https://fluxcd.io/), to enjoy all the benefits that come with it; and finally
+- as a StarlingX Application, which benefits from tight integration with the
+[StarlingX system](https://opendev.org/starlingx/config).
 
 ## Deploying PoC-StarlingX as a StarlingX App
 
-1. After packaging the app, send the generated `poc-starlingx-stx-pkg.tar.gz`
-file to the StarlingX active controller(s) where you want the app(s) to run. Then you
-can simply run:
-
+1. After [packaging the app](AppGenGuide.md), send the generated
```

Comment:
Readding a link from the Demo to (one of) the Build Guide (in this case, the one that uses the App Generator).

Reply:
ok